control_plane/src/compute.rs

```diff
@@ -284,14 +284,14 @@ impl PostgresNode {
     fn setup_pg_conf(&self, auth_type: AuthType) -> Result<()> {
         let mut conf = PostgresConf::new();
         conf.append("max_wal_senders", "10");
-        // wal_log_hints is mandatory when running against pageserver (see gh issue#192)
-        // TODO: is it possible to check wal_log_hints at pageserver side via XLOG_PARAMETER_CHANGE?
-        conf.append("wal_log_hints", "on");
+        // Wal logging hint bits may cause significant (>5 times) write amplification
```

Comment:
Is it an intentional part of the PR?

Reply:
It's ok. 
Just wanted to make sure that it is not an accidental change from some experiments.